template/src/common/text-area/text-area.module.scss

```
@@ -0,0 +1,105 @@
+@use 'sass:list';
+@use '../../assets/stylesheets/text-styles.scss';
+@import '../../assets/stylesheets/colors';
+
+$input-colors: (
+  default: ($text-neutral-20),
+  error: ($accent-danger-30),
+  success: ($accent-success-30),
+);
+$helper-colors: (
+  default: ($text-neutral-30),
+  error: ($accent-danger-30),
+  success: ($accent-success-30),
+);
+
+.inputContainer {
+  position: relative;
+  width: 100%;
+  display: flex;
+}
+
+.inputStyle {
+  padding: 12px;
+  width: 100%;
+  border: 1px solid $text-neutral-20;
+  border-radius: 10px;
+  transition: border, color 0.2s ease-in-out;
+  background-color: $surface-light-10;
+  color: $text-neutral-30;
+
+  @each $color, $properties in $input-colors {
+    &-#{$color} {
```

This might be a bit confusing since to give the input the success color you would need to add two classes to it: `inputStyles inputStyles-success`. I'd prefer if the color were a standalone class: `inputStyles success`
```suggestion
    &.#{$color} {
```